package.json

```diff
@@ -2,7 +2,7 @@
     "name": "monkey-c",
     "displayName": "Monkey C",
     "description": "Garmin Connect IQ Monkey C language support for VS Code",
-    "icon": "images/icon.svg",
+    "icon": "images/smart-monkey.png",
```

Comment:
what is the license for smart-monkey.png file? Is it free? CC?

Reply:
Good question got it from here: https://developer.garmin.com/connect-iq/programmers-guide/monkey-c;

Since I was not able to find any info, I will removed it.